src/cloudant/account.py

```diff
@@ -443,12 +448,14 @@ def volume_usage(self, year=None, month=None):
     def requests_usage(self, year=None, month=None):
         """
         Retrieves Cloudant requests usage data, optionally for a given
-        year/month.
+        year and month.
 
-        :param int year: Year to query against, for example 2014.  Defaults to
-            None.
+        :param int year: Year to query against, for example 2014.
+            Optional parameter.  Defaults to None.  If used, it must be
+            accompanied by the ``month``.
         :param int month: Month to query against, for example a number from
-            1 to 12.  Defaults to None.
+            1 to 12.  Optional parameter.  Defaults to None.
```

Comment:
"for example" isn't appropriate here, just use "must be an integer between 1 and 12".